.travis.yml

```diff
@@ -6,8 +6,24 @@ cache: cargo
 before_install:
   - rustup component add clippy
   - rustup component add rustfmt
+jobs:
+  include:
+    - rust: stable
+      script:
+        - cargo build --verbose
+        - cargo test --verbose
+    - rust: stable
+      script:
+        - cargo build --verbose --features "chrono"
+        - cargo test --verbose --features "chrono"
+    - rust: 1.38.0
+      script:
+        - cargo build --verbose
+        - cargo test --verbose
+    - rust: 1.38.0
+      script:
+        - cargo build --verbose --features "chrono"
+        - cargo test --verbose --features "chrono"
 script:
-  - cargo build --verbose
-  - cargo test --verbose
   - cargo clippy --verbose
   - cargo fmt && [ `git status -s | wc -l` -eq 0 ] # check that all files are formatted with `cargo fmt`
```

Comment:
it seems that this doesn't have the intended effect: if you check the travis builds you see that there are now 6 instead of 4 builds being triggered. the 4 defined in `jobs`, without the code in the standard `script` block plus one for rust 1.38 and one for rust stable each with the standard `script` block.

i'll try to have a look at this soon (i'm not deeply familiar with the build matrix configuration in travis either).

Reply:
Well, I did it like this to not copy clippy and format checks to all jobs. So four jobs build and test with and without the feature while two run checks which don't need feature variation. I think it's working as intended if my intended is the same as your intended of course.

I can just copy clippy and fmt to jobs, then there would be only 4 of them.

Reply:
see my other comment: this should do the trick (note: i took the liberty to change `$CHRONO` to `$FEATURES`, this allows for other features in the future):
```yml
language: rust
rust:
  - stable
  - 1.38.0  # lowest rust release against which we guarantee compatibility.
cache: cargo
env:
  - FEATURES='--features chrono'
  - FEATURES=''
before_install:
  - rustup component add clippy
  - rustup component add rustfmt
script:
  - cargo build --verbose $FEATURES
  - cargo test --verbose
  - cargo clippy --verbose
  - cargo fmt && [ `git status -s | wc -l` -eq 0 ] # check that all files are formatted with `cargo fmt`
```